filer/templates/admin/filer/folder/directory_listing.html

```diff
@@ -130,7 +130,7 @@
                             <div class="navigator-breadcrumbs-folder-name-wrapper">
                                 <span class="navigator-breadcrumbs-folder-name">
                                     <span class="navigator-breadcrumbs-folder-name-inner">
-                                        {% if folder.name != _("Unsorted Uploads") %}
+                                        {% if not folder.is_unsorted_uploads %}
                                             {{ folder.name }}
                                         {% else %}
                                             {% trans "Unsorted Uploads" %}
```

Comment:
actually, no need to even change it on the virtual Item. It already has the correct ``name`` attribute: https://github.com/divio/django-filer/blob/bugfix/rename-unfiled-files/filer/models/virtualitems.py#L50

The whole ``{% if .... %}...{% else %}...{% endif %}`` can be replaced by ``{{ folder.name }}``.